pubsublite/pscompat/subscriber.go

```diff
@@ -230,7 +230,7 @@ func (si *subscriberInstance) Wait(ctx context.Context) error {
 // is connected to multiple partitions. Only one call from any connected
 // partition will be outstanding at a time, and blocking in this receiver
 // callback will block the delivery of subsequent messages for the partition.
-type MessageReceiverFunc func(context.Context, *pubsub.Message)
+type MessageReceiverFunc = pubsub.MessageReceiverFunc
```

Comment:
If there are no objections to defining MessageReceiverFunc in the pubsub package, we could just remove this and change the parameter of SubscriberClient.Receive() to pubsub.MessageReceiverFunc. Any extra comments here can just be moved to SubscriberClient.Receive().

Reply:
Retracting, see comment below.